### PR TITLE
Supplier logic tests

### PR DESCRIPTION
**Pull request by KacFlor**

### CODE REVIEW

File: src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java [hunk 1]
```diff
@@ -23,8 +27,37 @@ public SupplierController(SupplierService supplierService){
...
     @GetMapping
-    public List<Supplier> getSupplier(){
-        return this.supplierService.getSupplier();
+    public List<Supplier> getAll(){
+        return this.supplierService.getAll();
+    }
+
+    @PreAuthorize("hasAnyAuthority('" + Role.Fields.USER + "', '" + Role.Fields.ADMIN + "')")
+    @GetMapping("/{id}")
+    public ResponseEntity<Optional<Supplier>> getById(@PathVariable("id") Integer id){
```
Comment (Ficzer) [inline src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java:38]: Make method names consistant as I explained to you

File: src/test/java/com/KacFlor/ShopSpring/controller/CardDataControllerTest.java [hunk 2]
```diff
@@ -108,4 +108,6 @@ void testDeleteByCustomerId() throws Exception{
                 .andExpect(status().isOk());
     }
 
+
+
```
Comment (Ficzer) [inline src/test/java/com/KacFlor/ShopSpring/controller/CardDataControllerTest.java:112]: delete

File: src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java (hunk 1 above)
Comment (KacFlor) [reply src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java:38]: done

File: src/test/java/com/KacFlor/ShopSpring/controller/CardDataControllerTest.java (hunk 2 above)
Comment (KacFlor) [reply src/test/java/com/KacFlor/ShopSpring/controller/CardDataControllerTest.java:112]: done

File: src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java (hunk 1 above)
Comment (Ficzer) [reply src/main/java/com/KacFlor/ShopSpring/controller/SupplierController.java:38]: I cann't see it, there is is still inconsistency

**Review by Ficzer — APPROVED** [review]

**Pull request merged**